core/templates/components/state-directives/hint-editor/hint-editor.component.ts

```diff
@@ -112,4 +118,4 @@ export class HintEditorComponent implements OnInit, OnDestroy {
 }
 
 angular.module('oppia').directive('oppiaHintEditor',
-  downgradeComponent({component: HintEditorComponent}));
+  downgradeComponent({ component: HintEditorComponent }));
```

Comment:
Why are these extra spaces being added?

Please check the "Files Changed" tab and drop any spurious changes from your PR.